explore/index.html

```diff
@@ -12,31 +12,118 @@
         <hr class="whitespace">
         <div class="row">
             <div class="medium-12 small-centered columns ">
-                <h4 class="subheader">OME products are in use in labs and
-                    research institutes around the world. Explore what they
-                    can do for you with this selection of demos and live
-                    online tools.</h4>
+                <h4 class="subheader">OME products are in use in labs and research institutes around the world. Explore what they can do for you with this selection of demos, live online tools and code sharing repositories.</h4>
```

Comment:
should this be "code-sharing repositories"? -- guessing that the "code sharing" is describing what the repositories are doing, rather than that it is code that is sharing repositories